licenses.d2iq.yaml

```diff
@@ -582,7 +582,7 @@ resources:
   - container_image: registry.k8s.io/ingress-nginx/kube-webhook-certgen:v20221220-controller-v1.5.1-58-g787ea74b6
     sources:
       - url: https://github.com/kubernetes/ingress-nginx
-        ref:  controller-1.5.2
```

Comment:
in last bump this was not modified tho? https://github.com/mesosphere/kommander-applications/pull/2537/files

Reply:
ok GET https://api.github.com/repos/kubernetes/ingress-nginx/contents//images/kube-webhook-certgen/rootfs?ref=controller-v1.5.2 actually works